game/src/LobbyMenu/LobbyMenu.gd

```diff
@@ -0,0 +1,27 @@
+extends Control
+
+signal back_button_pressed
+signal save_game_selected
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+func _on_back_button_button_down():
+	print("Returning to Main Menu.")
+	back_button_pressed.emit()
+
+
+func _on_start_button_button_down():
+	print("Starting new game.")
+	get_tree().change_scene_to_file("res://src/SampleGame.tscn")
```

Comment:
Prefer an export variable to a hardcoded scene file path. Example:
```gdscript
@export
var game_scene : PackedScene
# ...
get_tree().change_scene_to_packed(game_scene)
```

Reply:
For some reason using an export variable here caused the game to get stuck on the load screen, so I've left it out for now.

Reply:
I believe this is a case of godotengine/godot#73983, a regression introduced in Godot 4 RC 4, should be fixed in Godot 4 RC 6, I'll test it beforehand to make sure.